lib/features/details/presentation/view/movie_details_page.dart

```diff
@@ -72,6 +75,19 @@ class _MovieDetailsPageState extends State<MovieDetailsPage> {
                       fit: BoxFit.cover,
                     ),
                   ),
+                  child: IconButton(
```

Comment:
Can you make this play button just as the one for adding a movie to watch list?
Also put both buttons in the same row